data/registry_model/registry_oci_model.py

```diff
@@ -350,6 +351,14 @@ def create_manifest_and_retarget_tag(
         raise_on_error is set to True, in which case a CreateManifestException may also be
         raised.
         """
+        if features.QUOTA_MANAGEMENT:
```

Comment:
This will likely have unintented behavior due to wide spread usage of this function. Would need to have through tests built around it to ensure functionality that should work even above quota still does.

Reply:
Can you elaborate on this function's wide spread usage?
I'm only seeing it used in 2 places (beyond tests):
* endpoints/v2/manifest.py
* endpoints/v1/tag.py (indirectly via `data.registry_model.manifestbuilder.commit_tag_and_manifest`